commit 61b5194808f405fbf901f7c8280333cad0ec9768
Author: Nadav Amit <namit@vmware.com>
Date:   Thu Sep 13 13:18:52 2018 -0700

    vmw_balloon: include asm/io.h
    
    commit a3b92ee6fc171d7c9d9b6b829b7fef169210440c upstream.
    
    Fix a build error due to missing virt_to_phys()
    
    Reported-by: kbuild test robot <lkp@intel.com>
    Fixes: f0a1bf29d821b ("vmw_balloon: fix inflation with batching")
    Cc: stable@vger.kernel.org
    Cc: Xavier Deguillard <xdeguillard@vmware.com>
    Signed-off-by: Nadav Amit <namit@vmware.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/misc/vmw_balloon.c b/drivers/misc/vmw_balloon.c
index 5f8b583c6e41..f74166aa9a0d 100644
--- a/drivers/misc/vmw_balloon.c
+++ b/drivers/misc/vmw_balloon.c
@@ -45,6 +45,7 @@
 #include <linux/seq_file.h>
 #include <linux/vmw_vmci_defs.h>
 #include <linux/vmw_vmci_api.h>
+#include <linux/io.h>
 #include <asm/hypervisor.h>
 
 MODULE_AUTHOR("VMware, Inc.");
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1085_61b5194808f405fbf901f7c8280333cad0ec9768.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1085_61b5194808f405fbf901f7c8280333cad0ec9768.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
